doc/apple2.sgml

```diff
@@ -511,6 +511,109 @@ url="ca65.html" name="assembler manual">.
 </descrip><p>
 
 
+<sect1>Specifying file types for fopen<p>
+
+<descrip>
+
+  <tag>Explanation of File Types</tag>
+
+  ProDOS associates a file type and an auxiliary type with each file.
+  These type specifications are separate from the file's name, unlike
+  Windows and UNIX-like systems which use the file name's suffix (a.k.a.
+  extension) to specify the file type. For example, .exe, .doc, or .bat.
```

Comment:
Extensions in UNIX are not required or used by the system, but some programs do use them: gcc, make, java, Eclipse, vim, gnotepad. Make in particular has implicit rules that are based on file extensions. Although you could say they are just file name patterns, rather than extensions. The patterns though usually have to do with the suffix of the file. The "file" command does look at the contents of the file, rather than its extension, to determine it's type. Regardless I can remove the reference to UNIX. It's not that important.